plugins/outputs/dynatrace/dynatrace.go

```diff
@@ -236,7 +236,7 @@ func (d *Dynatrace) send(msg []byte) error {
 	req, err := http.NewRequest("POST", d.URL, bytes.NewBuffer(msg))
 	if err != nil {
 		d.Log.Errorf("Dynatrace error: %s", err.Error())
-		return fmt.Errorf("Dynatrace error while creating HTTP request:, %s", err.Error())
+		return fmt.Errorf("dynatrace error while creating HTTP request:, %s", err.Error())
```

Comment:
Maybe remove `dynatrace` completely or otherwise leave capitalized as it is a name.

Reply:
Removed